src/app/components/loaders/stacks-account-loader.tsx

```diff
@@ -16,6 +17,10 @@ export function CurrentStacksAccountLoader({
   fallback,
 }: CurrentStacksAccountLoaderProps) {
   const currentAccount = useCurrentStacksAccount();
+  const isLedger = useHasLedgerKeys();
+
+  if (!currentAccount && !isLedger) return null;
+
```

Comment:
I don't think the loader should ever return something other than child or fallback.

Maybe this should be fixed in the instance of `CurrentStacksAccountLoader` and not inside of it?